ui/android/BUILD.gn

```diff
@@ -240,13 +240,23 @@ android_library("ui_utils_java") {
     "java/src/org/chromium/ui/KeyboardVisibilityDelegate.java",
     "java/src/org/chromium/ui/MotionEventUtils.java",
     "java/src/org/chromium/ui/UiUtils.java",
+    "java/src/org/chromium/ui/display/DisplaySwitches.java",
+    "java/src/org/chromium/ui/display/DisplayAndroid.java",
+    "java/src/org/chromium/ui/display/DisplayAndroidManager.java",
+    "java/src/org/chromium/ui/display/PhysicalDisplayAndroid.java",
   ]
```

Comment:
Factor it out so that upstream / Chromium build configs don't change

On line 247
```
if(is_cobalt) {
 sources += [
   ...
 ]
}
```